tools/add_actor

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env python
+""" Script to gather information and create a new actor using Snactor """
+import os
+import sys
+from string import Template
+
+
+YAML_TEMPL = r"""outputs:
```

Comment:
Hi, I believe it will be better to keep the templates separated in their own files outside of the script. This way you can simply extend the template without touching the scripts. Also, later an option with path to own template can be added.